src/View.Uno/Extensions/ControlAutoFocus/ControlExtensions.AutoFocus.cs

```diff
@@ -145,7 +145,17 @@ private static void IsAutoFocusChanged(DependencyObject d, DependencyPropertyCha
 		private static void OnLoaded(object sender, RoutedEventArgs e)
 		{
 			var control = sender as Control;
-			ApplyAutoFocus(control);
+#if !__IOS__ && !__MACOS__
+			ApplyAutoFocus(control);			
+#else
+			// When UITextField is focused right after it is moved to window,
+			// the first responder request will succeed, but the keyboard will
+			// not be displayed - we must delay the request slightly.
```

Comment:
Let's add a reference to the uno issue as well, thanks!

Reply:
Done 👍